test/RESTResource.js

```diff
@@ -43,6 +44,11 @@ describe('RESTResource', () => {
         .should.equal('/whatever/water/anyways');
     });
 
+    it('replaces header values', () => {
```

Comment:
This test is fine so far as it goes, but it's a bit weak. Can you easily test the automatic expansion of `headers`?

Reply:
I agree @MikeTaylor, it **is** weak! I've spent a while trying to figure out what a better test would look like and have, so far, failed. The current tests only test `substitute` and the changes I made are in `verbOptions`. I think I'm not 100% clear on what you mean by "automatic expansion of headers", sorry if I'm being dim!

I intend creating tests for my new `buildOption` method as it needs some and Sonarcloud is complaining.

Reply:
OK, well you shouldn't worry too much about that. Testing is a means to an end.

Reply:
Thanks @skomorokh You're absolutely right, the test naming was completely inaccurate! I've not modified it accordingly and moved it into its own commit. Thanks for the feedback.